lib/src/components/NxNavigationSidebar/NxNavigationSidebarContent.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React, { FunctionComponent } from 'react';
+import classnames from 'classnames';
+
+import { NxNavigationSidebarContentProps as Props, nxNavigationSidebarContentPropTypes } from './types';
+export { NxNavigationSidebarContentProps } from './types';
+
+const NxNavigationSidebarContent: FunctionComponent<Props> = function NxNavigationSidebarContent(props) {
+  const { className, children } = props;
```

Comment:
Rather than only allowing `className` and `children`, what you probably want to do is allow any props that a `section` can take, and just have some logic deal with the classnames.

Reply:
> Rather than only allowing `className` and `children`, what you probably want to do is allow any props that a `section` can take, and just have some logic deal with the classnames.

Like this? 7edb99c246614a4561d9bdac48c7a15f43f07c87

Reply:
> Like this? 7edb99c

More or less.  You don't even need to pull out `children` though

Reply:
> > Like this? [7edb99c](https://github.com/sonatype/sonatype-react-shared-components/commit/7edb99c246614a4561d9bdac48c7a15f43f07c87)
> 
> More or less. You don't even need to pull out `children` though

Like this? ba0b3b26b4170f3b8360e7e342c33b8936f011e2